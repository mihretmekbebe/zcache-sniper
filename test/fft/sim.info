{'cmdline': ['../../run-sniper',
             '-n',
             '2',
             '-c',
             'gainestown',
             '--roi',
             '--viz',
             '--',
             './fft',
             '-p',
             '2'],
 'git_revision': '',
 'host': 'llamaa',
 'pin_version': 71313,
 'rusage': (5.676215,
            0.6597069999999999,
            61116,
            0,
            0,
            0,
            142376,
            22,
            0,
            5200,
            4416,
            0,
            0,
            0,
            10795,
            47),
 'snipercmd': ['bash',
               '-c',
               '/home/bragadeesh153/Desktop/Sniper/sniper-6.1/pin_kit/intel64/bin/pinbin -mt -injection child -ifeellucky -xyzzy -enable_vsm 0 -t /home/bragadeesh153/Desktop/Sniper/sniper-6.1/lib/pin_sim -c /home/bragadeesh153/Desktop/Sniper/sniper-6.1/config/base.cfg --general/total_cores=2 --general/output_dir=/home/bragadeesh153/Desktop/Sniper/sniper-6.1/test/fft --config=/home/bragadeesh153/Desktop/Sniper/sniper-6.1/config/nehalem.cfg --config=/home/bragadeesh153/Desktop/Sniper/sniper-6.1/config/gainestown.cfg -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/bragadeesh153/Desktop/Sniper/sniper-6.1/test/fft/sim.scripts.py -g --hooks/script0args= -- ./fft -p 2'],
 't_elapsed': 5.448828935623169,
 't_start': 1531474419.183757,
 'tracecmds': [],
 'user': 'bragadeesh153',
 'vmem': 40046592L}
