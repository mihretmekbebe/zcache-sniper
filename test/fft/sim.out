                                     | Core 0     | Core 1    
  Instructions                       |     568603 |     569723
  Cycles                             |     695493 |     695493
  IPC                                |       0.82 |       0.82
  Time (ns)                          |     261464 |     261464
  Idle time (ns)                     |      93429 |      98548
  Idle time (%)                      |      35.7% |      37.7%
Branch predictor stats               |            |           
  num correct                        |      15691 |      15730
  num incorrect                      |       1010 |       1074
  misprediction rate                 |      6.05% |      6.39%
  mpki                               |       1.78 |       1.89
TLB Summary                          |            |           
  I-TLB                              |            |           
    num accesses                     |      38330 |      38580
    num misses                       |          5 |         33
    miss rate                        |      0.01% |      0.09%
    mpki                             |       0.01 |       0.06
  D-TLB                              |            |           
    num accesses                     |     333030 |     333237
    num misses                       |          8 |         32
    miss rate                        |      0.00% |      0.01%
    mpki                             |       0.01 |       0.06
  L2 TLB                             |            |           
    num accesses                     |         13 |         65
    num misses                       |         13 |         65
    miss rate                        |    100.00% |    100.00%
    mpki                             |       0.02 |       0.11
Cache Summary                        |            |           
  Cache L1-I                         |            |           
    num cache accesses               |      38330 |      38580
    num cache misses                 |        254 |        338
    miss rate                        |      0.66% |      0.88%
    mpki                             |       0.45 |       0.59
  Cache L1-D                         |            |           
    num cache accesses               |     333142 |     333329
    num cache misses                 |        649 |        787
    miss rate                        |      0.19% |      0.24%
    mpki                             |       1.14 |       1.38
  Cache L2                           |            |           
    num cache accesses               |       1158 |       1319
    num cache misses                 |        497 |       1069
    miss rate                        |     42.92% |     81.05%
    mpki                             |       0.87 |       1.88
  Cache L3                           |            |           
    num cache accesses               |        766 |       1265
    num cache misses                 |        344 |        362
    miss rate                        |     44.91% |     28.62%
    mpki                             |       0.60 |       0.64
DRAM summary                         |            |           
  num dram accesses                  |        705 |          0
  average dram access latency (ns)   |      75.40 |        inf
  average dram queueing delay        |      21.97 |       0.00
  average dram bandwidth utilization |      2.00% |      0.00%
Coherency Traffic                    |            |           
  num loads from dram                |        133 |        140
  num loads from dram cache          |          0 |          0
  num loads from remote cache        |          0 |          0
